src/main/java/frc/robot/RobotMap.java

```diff
@@ -37,4 +38,9 @@ public static final class mapIntake {
     public static final int LEFT_MOTOR_CAN = 11;
     public static final int RIGHT_MOTOR_CAN = 12;
   }
+
+  public static final class mapUtilityArm {
+    public static final int PIVOT_MOTOR = 0;
+    public static final int INTAKE_MOTOR = 1;
+  }
```

Comment:
Don't forget to put the connection type for the mapping. Also 0 and 1 are already used for DT motors so we must change it